docs/tutorial/beginner.rst

```diff
@@ -240,91 +301,75 @@ What Your Directory Structure Should Like After This Run:
 
 .. code-block:: yaml
     
     run2:   
         k: [10, 100] 
 
-With this update, the beginner.yaml configuration file is set up have SPRAS run a single algorithm with multiple parameter settings on one dataset.
+With this update, the ``beginner.yaml`` configuration file is set up have SPRAS run a single algorithm with multiple parameter settings on one dataset.
 
 After saving the changes, rerun with:
 
 .. code:: bash
 
     snakemake --cores 1 --configfile config/beginner.yaml
 
-What Happens When You Run This Command
+What happens when you run this command
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 1.	Snakemake loads the configuration file
 
-Snakemake reads beginner.yaml to determine which datasets, algorithms, parameters, and post-analyses to run. 
+Snakemake again reads ``beginner.yaml`` to determine which datasets, algorithms, parameters, and post-analyses to run. 
+
 It reuses cached results to skip completed steps, rerunning only those that are new or outdated. 
-Here, the dataset pickle, PathLinker inputs, and D4TUKMX parameter set are reused instead of rerun.
+Here, the Pathlinker prepared inputs are reused.
 
 2. Organizing outputs per parameter combination
 
-Each new dataset-algorithm-parameter combination gets its own folder (e.g egfr-pathlinker-params-7S4SLU6/ and egfr-pathlinker-params-VQL7BDZ/)
-The hashes 7S4SLU6 and VQL7BDZ uniquely identifies the specific set of parameters used.
+Each new <dataset>-<algorithm>-params-<hash> combination gets its own folder created in ``output/basic/``.
+
+A matching log file is placed in ``logs/parameters-<dataset>-params-<hash>.yaml`` which records the exact parameter value used.
 
 3. Reusing prepared inputs with additional parameter combinations
 
-Since PathLinker has already been run once, SPRAS uses the cached prepared inputs (network.txt, nodetypes.txt) rather than regenerating them.
-For each new parameter combination, SPRAS executes the PathLinker by launching its corresponding Docker image multiple times (once for each parameter configuration). 
-PathLinker then runs and produces a raw-pathway.txt file specific to each parameter hash.
+For each new parameter combination and its corresponding cached prepared inputs, SPRAS executes PathLinker by launching multiple Docker contatiners (once for each parameter configuration). 
+
+PathLinker then runs and produces a ``raw-pathway.txt`` file specific to each parameter and places it in it's corresponding folder.
 
 4. Parsing into standardized results
 
-SPRAS parses each new raw-pathway.txt file into a standardized SPRAS format (pathway.txt).
+SPRAS parses each new ``raw-pathway.txt`` file into a standardized SPRAS format (``pathway.txt``) and places it in it's corresponding folder.
 
-What Your Directory Structure Should Like After This Run:
+5. Logging the Snakemake run 
+
+Snakemake creates a dated log in ``.snakemake/log/``.
```

Comment:
[Do we want to describe what the structure of this is like?]

Reply:
I don't think it's important.